session.go

```diff
@@ -591,6 +592,9 @@ func CreateSession(store kv.Storage) (Session, error) {
 	variable.BindSessionVars(s)
 	variable.GetSessionVars(s).SetStatusFlag(mysql.ServerStatusAutocommit, true)
 
+	// set connection id
+	s.SetValue(builtin.ConnectionIDKey, s.sid)
```

Comment:
Returns the connection ID (thread ID) for the connection. Every connection has an ID that is unique among the set of currently connected clients.
SessionID is unique among the clients.